checker/check.go

```diff
@@ -87,7 +86,11 @@ func (triggerChecker *TriggerChecker) handlePrepareError(checkData moira.CheckDa
 func (triggerChecker *TriggerChecker) handleFetchError(checkData moira.CheckData, err error) error {
 	switch err.(type) {
 	case ErrTriggerHasEmptyTargets, ErrTriggerHasOnlyWildcards:
-		triggerChecker.logger.Debugf("Trigger %s: %s", triggerChecker.triggerID, err.Error())
+		triggerChecker.logger.Debugb().
+			String("trigger_id", triggerChecker.triggerID).
+			Error(err).
+			Msg("Trigger fetch")
```

Comment:
i think be better

Trigger was fetched

Reply:
Fixed